_security/access-control/users-roles.md

```diff
@@ -137,7 +141,6 @@ For more detailed summaries of the permissions for each role, reference their ac
 
 The following examples show how you might set up a read-only and a bulk access role.
```

Comment:
Suggestion: ...a read-only role and a bulk access role. OR ...read-only and bulk access roles.